src/sweetalert2.all.js

```diff
@@ -1,4 +1,4 @@
-import styles from '../dist/sweetalert2.css'
+import styles from './sweetalert2.scss'
```

Comment:
Bear in mind this won't autoprefix the styles like the gulp task does, hence why I made it pull in the dist. We either have to pull in the dist or duplicate the sass build pipline in rollup and gulp

Reply:
Thanks for the review @samturrell . Yes, this makes sense. I seemed to have a (maybe impractical) issue with the src depending on the dist. In my next go at this PR I will change this line back but provide `sweetalert2.css` from a Vinyl (Gulp) stream rather than from the dist folder.